web/source/dom/domEventHandlers.ts

```diff
@@ -423,11 +423,13 @@ namespace com.keyman.dom {
       } else if(el && el.className.indexOf('kmw-disabled') >= 0) {
         return true; 
       }
-      
+
+      /*
       // Or if OSK not yet ready (for any reason)
-      if(!osk.ready) {
+      if(!osk) {
         return true;
       }
+      */
```

Comment:
This is commented out -- should it be removed?

Reply:
That's... actually a good question.

The prior behavior here:  if the OSK wasn't initialized, KeymanWeb would block use of the active keyboard for keystroke processing, probably as a failsafe of some sort.  I don't believe this to be appropriate any longer, so I'd advocate for removal.  (It covered an 'exceptional' case that shouldn't have ever occurred during actual use.)

There's definitely room for discussion, hence why I left it in.

Reply:
Sounds to me like it should be stripped out.